courses/free-courses-it.md

```diff
@@ -126,6 +126,7 @@
 ### Java
 
 * [Algoritmi e Strutture Dati Java](http://www.vitoantoniobevilacqua.it/wiki/index.php?title=Pagina_principale#Dispense_di_Algoritmi_e_Strutture_Dati_in_Java) - V. Bevilacqua (Politecnico di Bari)  
+* [Corso di Java Spring Framework](https://www.youtube.com/playlist?list=PLCbSCJEIR6CqgCLyVzqp49xOm8A5YDTKA) (M. Cicolella)
```

Comment:
Use right resources format for courses: `TITLE - INSTRUCTOR LIST (Platform) *(Notes)*`

```suggestion
* [Corso di Java Spring Framework](https://www.youtube.com/playlist?list=PLCbSCJEIR6CqgCLyVzqp49xOm8A5YDTKA) - Mauro Cicolella
```